Intel_SighPP_CMake/Intel_SighPP_CMake/src/priority_lib/covid_priority.hpp

```diff
@@ -0,0 +1,158 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Sight++. All Rights Reserved.
+
+#pragma once
+
+#include "priority_module.hpp"
+#include "priority_clock.hpp"
+#include <spdlog/spdlog.h>
+#include "../classification_result.hpp"
+#include <string>
+#include <map>
+#include <set>
+
+class covid_priority : public priority_module
+{
+protected:
+	int size_w = 640;
+	int size_h = 480;
+	int const cooldown = 7000;
+	int const cooldown_shortening = 1000;
+	int const cooldown_emergency = 3000;
+	double const walking_speed = 1.4;
+	int centre_line = 0;
+	int out_left = 0;
+	int out_right = 0;
+	int in_left = 0;
+	int in_right = 0;
+	int above = 0;
+	int bottom = 0;
+	int mid_w = 0;
+	priority_clock pc;
+	Priority min = Priority::UNDEFINED;
+	std::set<std::string> covid_list{"Person","person","bycycle","Bycycle"};
+
+public:
+
+	/// <summary>
+	/// Object Creation. Not to be inherited furhter.
+	/// Initialises location markers based on dividers, mostly in fifths of the screen.
+	/// </summary>
+	/// <param name="name"></param>
+	/// <returns></returns>
+	covid_priority(std::string name = "default")
```

Comment:
What does name do? Maybe explain it or have a better one than default as the default 😄